server/src/jobs/oneshot/qualiopi/index.js

```diff
@@ -31,7 +31,7 @@ runScript(async ({ db }) => {
 
   // Formations
   Formation.pauseAllMongoosaticHooks();
-  await trainingsUpdater({ noUpdatesFilters: true, noMail: true });
+  await trainingsUpdater({ noMail: true });
```

Comment:
```suggestion
  await trainingsUpdater({ noUpdatesFilters: true, noMail: true });
```

Pour le job qualiopi le paramètre `noUpdatesFilters` garantit que ça va s’exécuter sur toutes les formations.

Reply:
Corrigé